application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1132,7 +1191,7 @@ type QueryType {
     routes: [String],
     "Trip gtfsIds (e.g. [\"HSL:1098_20190405_Ma_2_1455\"])."
     trips: [String]
-  ): [Stoptime]
+  ): [Stoptime] @deprecated(reason : "`cancelledTripTimes` is deprecated. Use `canceledTrips` instead.")
```

Comment:
I think the deprecation message should state clearly that it was never implemented.

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/5393/commits/7ed48c3d556ab37bb375f8c23eb34018c30d5b39